client/src/common/protocolConverter.ts

```diff
@@ -1088,6 +1088,8 @@ export function createConverter(uriConverter: URIConverter | undefined, trustMar
 					for (const edit of change.edits) {
 						if (ls.AnnotatedTextEdit.is(edit)) {
 							result.replace(uri, asRange(edit.range), edit.newText, asMetadata(edit.annotationId));
+						} else if (ls.SnippetTextEdit.is(edit)) {
+							result.replace(uri, asRange(edit.range), edit.snippet.value, asMetadata(edit.annotationId));
```

Comment:
I don't know anything about vscode but wouldn't this need to handle snippet text in some special way? I would imagine that it would otherwise insert a raw, un-expanded snippet value?

Reply:
That's right, such edit should be parsed as a snippet and expanded accordingly by the editor. The capability for this feature should ensure that the editor expects these "special snippet strings".

Reply:
@MariaSolOs @rchl is actually correct and I totally missed that as well. This loop https://github.com/microsoft/vscode-languageserver-node/blob/2c7e6238d7fff8198f1bc0872ce46c0d40eaa6e7/client/src/common/protocolConverter.ts#L1088 needs to be recoded. I need to talk for Joh first since we currently can't mix edits with or without meta data when using the set method. 

I would opt that we disable the capability for now until we know how to actually do this correctly. @MariaSolOs I will do that.

Reply:
@MariaSolOs : we need to add VS Code SnippetTextEdit to the workspace edit in the snippet cases.